LGTM.

Perhaps also worth mentioning currently the GTK WebDriver post-commit bot is failing due to this bug.

https://build.webkit.org/#/builders/37/builds/13984

```
Traceback (most recent call last):
  File "/app/webkit/Tools/Scripts/run-webdriver-tests", line 85, in <module>
    runner.run(args)
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/webdriver_test_runner.py", line 70, in run
    runner_tests = [runner.collect_tests(tests) for runner in self._runners]
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/webdriver_test_runner.py", line 70, in <listcomp>
    runner_tests = [runner.collect_tests(tests) for runner in self._runners]
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/webdriver_test_runner_selenium.py", line 49, in collect_tests
    executor = WebDriverSeleniumExecutor(self._driver, self._env)
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/webdriver_selenium_executor.py", line 52, in __init__
    do_delayed_imports()
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/webdriver_selenium_executor.py", line 30, in do_delayed_imports
    import webkitpy.webdriver_tests.pytest_runner as pytest_runner
  File "/app/webkit/Tools/Scripts/webkitpy/webdriver_tests/pytest_runner.py", line 32, in <module>
    import pytest
  File "/app/webkit/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/pytest/__init__.py", line 5, in <module>
    from _pytest._code import ExceptionInfo
  File "/app/webkit/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/_pytest/_code/__init__.py", line 2, in <module>
    from .code import Code
  File "/app/webkit/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/_pytest/_code/code.py", line 60, in <module>
    from exceptiongroup import BaseExceptionGroup
ModuleNotFoundError: No module named 'exceptiongroup'
```

--- a/Tools/Scripts/webkitpy/__init__.py
+++ b/Tools/Scripts/webkitpy/__init__.py
@@ -40,6 +40,8 @@ if sys.version_info >= (3, 7):
     AutoInstall.register(Package('pytest_timeout', Version(2, 1, 0), pypi_name='pytest-timeout'))
     AutoInstall.register(Package('pytest', Version(7, 2, 0), implicit_deps=['pytest_asyncio', 'pytest_timeout']))
     AutoInstall.register(Package('websockets', Version(8, 1)))
+    if sys.version_info < (3, 11):
+        AutoInstall.register(Package('exceptiongroup', Version(1, 1, 0), wheel=True))
 elif sys.version_info >= (2, 7) and sys.version_info < (3,):
     AutoInstall.register(Package('pylint', Version(0, 28, 0)))
     AutoInstall.register(Package('logilab.common', Version(0, 58, 1), pypi_name='logilab-common', aliases=['logilab']))

